### PR TITLE
check for both content and message

### PR DESCRIPTION
**Pull request by tparikh**
Signed-off-by: Tejas Parikh <tparikh@redhat.com>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -72,10 +77,26 @@ func sendToCloudWatch(logPart format.LogParts) {
...
 
+	// rfc5424
+	m := logPart["message"]
+
+	// rfc3164
+	if m == nil {
+		m = logPart["content"]
+	}
+
+	log.Println(logPart)
```
Comment (jakedt) [inline main.go:88]: Won't this be too noisy?

File: main.go [hunk 2]
```diff
@@ -72,10 +77,26 @@ func sendToCloudWatch(logPart format.LogParts) {
...
+
+	if m == nil {
+		return
+	}
+
 	params := &cloudwatchlogs.PutLogEventsInput{
 		LogEvents: []*cloudwatchlogs.InputLogEvent{
 			{
-				Message:   aws.String(logPart["message"].(string)),
+				Message:   aws.String(m.(string)),
```
Comment (jakedt) [inline main.go:99]: This can panic.

**Review by jakedt — COMMENTED** [review]
We should also check for an error on everything that can error.

File: main.go (hunk 1 above)
Comment (tparikh) [reply main.go:88]: Yes. I will update this later today. This will help with debug today.

File: main.go (hunk 2 above)
Comment (tparikh) [reply main.go:99]: It will recover from panic.

**Pull request merged**